main.html

```diff
@@ -1,77 +1,255 @@
-    F2 = cos(2 * pi * 2 * bl_length * l / wavelength)
-
-    plt.plot(l, F, c='#cc0000', label=&quot;Baseline 1-2&quot;)
-    plt.plot(l, F2, c='#0000cc', label=&quot;Baseline 1-3&quot;)
-    plt.xlabel(&quot;$sin(\\theta)$&quot;)
-    plt.ylabel(&quot;Fringe amplitude&quot;)
-    plt.ylim(-2, 2)
-    plt.legend()
-
-f = interact(plot_fringe, bl_length=(1, 100), wavelength=(1, 100))
-&lt;/pre&gt;
+<p>
+  In this Oriole, we'll define some key terms used when discussing InnerSource.
+  We'll walk through the high-level mechanics of how InnerSourcing can work and enumerate the key principles upon-which these mechanics are based.
+  Finally, we'll share other references that can help you deepen your understanding and involvement in the craft of InnerSource.
+</p>
+
+<h2>Context</h2>
+
+<p>
+  Imagine two teams delivering separate pieces of software with one team's software depending on that of the other.
+  This situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+  When consuming teams need many features, producing teams normally have some sort of requirements and prioritization process to decide which features they will work on.
+  For feature requests that are not prioritized, the consuming team is faced with 3 choices, each of which come with their own drawbacks.
+</p>
+
+<ol>
+  <li>
+    <p>
+      <strong>Wait it out</strong>.
+      In this option, the consuming team does nothing and limps along without the requested functionality.
+      Depending on the benefit of the feature request, <em>Wait it out</em> might be just fine or it may come with real amounts of pain for the consuming team. 
+    </p>
+  </li>
+  <li>
+    <p>
+      <strong>Workaround</strong>.
+      A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
+      This extra work may come as a modification (hack) in the consuming project.
+      Alternatively, the consuming team may create a new project that meets their needs and replaces their use of all or part of the producing team's system (code/project duplication).
+    </p>
+    <p>
+      <em>Workaround</em> is suboptimal from a variety of standpoints.
+      <ul>
+        <li>
+          First, any work done by the consuming team remains unavailable to any other consumers with the same feature request.
+        </li>
+        <li>
+          Second, the consuming team has inadvertently signed up for the long-term burden of maintaining the newly-written code, which is not in their core business domain.
+        </li>
+        <li>
+          Third, the company as-a-whole acquires duplicate projects and code in the same problem space.
+        </li>
+      </ul>
+    </p>
+    <p>
+      Despite these drawbacks, one big win for the consuming team is their ability to get the requested feature via their own efforts only (no hard dependencies).
```

Comment:
I think you should put it first because if this is the most common result of the **Workaround** scenario then it should be highlighted first rather than at the end.  

In thinking about this more I think all 3 scenarios could have a little more structure to them to more clearly present the material.  

Features, Advantages(or disadvantages), Benefits.
[This article](http://devedge-internet-marketing.com/2012/10/06/features-advantages-and-benefits-fab-statements/) goes into more detail.